### ISSUE TITLE
new cover images not picked up on scan

### ISSUE THREAD

**Issue by dicktyr** [squentin/gmusicbrowser#100]
I recently populated my library with cover.jpg files
but the startup scans fail to find them

is there any way to rescan for cover images
without reimporting the whole library?

**Comment by squentin** [comment]
Yes, though it's currently a bit hidden, you have to select more than 1 albums in the album list, then you will get an "Auto-select pictures" entry in their context menu. Note that will only try to find a picture for albums that have no picture set, or whose picture file is no longer there.

I plan to add this to the scan or the check, I'll close this bug when it's done.

**Comment by remicollet** [comment]
Notice: album cover is not always saved.

Probably only saved during a proper "quit". But as I always keep gmusicbrowser open (launch by session manager, close at session close), perhaps it need to catch this event.

**Comment by jplatte** [comment]
@remicollet There's a plugin that automatically saves every X minutes. Would be nice if `SIGTERM` was caught and the config file was saved as well then, but that should probably be a seperate issue.
